Gruntfile.js

```diff
@@ -134,6 +134,7 @@ module.exports = function(grunt) {
 
     const webpackDevConfig = Object.assign({}, webpackConfig, {
         resolve: {
+            modules: [path.join(__dirname, 'app/scripts'), path.join(__dirname, 'node_modules')],
```

Comment:
Since this line is equal, probably it's better to assign only alias to a new object, to avoid repeating this code here.